scripts/functions/requirements/osx_brew

```diff
@@ -541,6 +541,11 @@ requirements_osx_brew_ensure_brew_can_install()
   \typeset __celar_path
   __celar_path="$(brew --cellar)"
   if
+    [[ -x /opt/workbrew/bin/brew ]]
```

Comment:
`brew --cellar` still returns `/opt/homebrew/Cellar`, unfortunately. The presence of this file should be sufficient. Some other options:
- check if the `workbrew` owner owns `/opt/homebrew/bin/brew` or `/opt/workbrew/bin/brew`
- check for `/opt/workbrew/bin/brew --version` output to contain `Workbrew`
- prioritise `/opt/workbrew/bin/brew` over `/opt/homebrew/bin/brew` or `brew` in `PATH` if it exists

Reply:
Shouldn't we rather ensure, that when rvm calls `brew` it would actually call `brew` coming from `workbrew` and not the _standard one_? I assume user could potentially have both installed? Shall we than check it `which brew` points to `/opt/workbrew/bin/brew`?

Reply:
@pkuczynski Apologies that this may be a little confusing to explain:

When Workbrew is installed: `/opt/homebrew/bin/brew` still exists as a normal, unmodified Homebrew installation, it's just owned by the `workbrew` user instead of the current one. `/opt/workbrew/bin/brew` provides a wrapper which is responsible for (among other things) changing user to `workbrew` to execute Homebrew.

If `/opt/workbrew/bin/brew` is installed: things like `/opt/homebrew/bin/brew shellenv` will add `/opt/workbrew/bin/brew` to the front of the path.

As a result, it's a (very) safe bet that if `/opt/workbrew/bin/brew` exists: it's expected that `/opt/homebrew/bin/brew` will never be writable to the current user. If somehow it's owned by a non-current-user, non-workbrew-user setup: it'll fail to install later and that'll provide debugging for users at that point.

Does that make sense?

(if it helps for context: I'm the Project Leader and longest tenured maintainer of Homebrew and CTO and Cofounder of Workbrew)

Reply:
It does make sense I think. In principle, I don't have a problem with your PR as I said before :) So add a CHANGLOG and we will merge it in.

PS: I know who you are :) Great work with homebrew btw! :) And since I have you here, how can we get `rvm` to brew? Sounds like a good idea overall.

Reply:
@pkuczynski Ok, thanks.

> So add a CHANGLOG and we will merge it in.

Does this mean:
- add a new line to https://github.com/rvm/rvm/blob/master/CHANGELOG.md (under "bug fixes", presumably?)
- commit it to this PR (in a new commit or same commit or don't care?)

Reply:
1. Yes, 2. We are squashing so can be a new commit.

Reply:
@pkuczynski Pushed now (in existing commit)